tests/ballerina-test/src/test/java/org/ballerinalang/test/nativeimpl/functions/io/bytes/BytesInputOutputBufferTest.java

```diff
@@ -85,29 +170,32 @@ public void varyingBufferSizeTest() throws IOException, URISyntaxException {
         final int fixedBufferSize = 15;
```

Comment:
Unused variable?

Reply:
Kept this for testing purposes when async function is available. Since it's not 
Removed the variable with 3b58422f6c457c12c1ed16661b17ae49a52a17a7 
Will add this once it's available.